### PR TITLE
@zamiang/@craigspaeth Callout Admin

### PR DESCRIPTION
**Pull request by kanaabe**
This one was tricky!

General callout functionality, ability to override title and thumbnail: 
callout

Only show callout option if it's above a text section: 
callout2

Pull Quote: 

@owendodd I'd like to swap out the callout icon!

### CODE REVIEW

**Comment by owendodd** [comment]
So cool :) 

Will send you the svg in the AM!

**Comment by craigspaeth** [comment]
image

File: client/apps/edit/components/section_callout/index.coffee [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+_ = require 'underscore'
+React = require 'react'
+sd = require('sharify').data
+gemup = require 'gemup'
+Autocomplete = require '../../../../components/autocomplete/index.coffee'
+{ div, section, input, a, h1, h2, button, img, p, strong, span } = React.DOM
+{ crop } = require('embedly-view-helpers')(sd.EMBEDLY_KEY)
+Article = require '../../../../models/article.coffee'
+
+module.exports = React.createClass
```
Comment (craigspaeth) [inline client/apps/edit/components/section_callout/index.coffee]: :clap: Well done with this component! It's like the dream team of components—`crop` `gemup` `Autocomplete` and tons of functionality of it's own.

File: client/apps/edit/components/section_callout/index.styl [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    width 100%
+    vertical-align middle
+    padding 10px 0 10px 0
+
+.esc-callout-right
+  max-width 300px
+  padding 20px 0 20px 0
+  text-align center
+
+.esc-autocomplete-input
```
Comment (craigspaeth) [inline client/apps/edit/components/section_callout/index.styl]: Flawless namespacing!

File: client/apps/edit/components/section_tool/index.coffee [hunk 3]
```diff
@@ -149,3 +160,12 @@ module.exports = React.createClass
                 className: 'edit-menu-icon-embed'
                 dangerouslySetInnerHTML: __html: $(icons()).filter('.embed').html()
               }
+          if sd.USER?.type is 'Admin' and @isAboveTextSection()
```
Comment (craigspaeth) [inline client/apps/edit/components/section_tool/index.coffee]: :100: very nice!

**Comment by craigspaeth** [comment]
Impressive work! Very cool feature and no complaints on code—clean, clear, well composed, and solid test coverage!

**Pull request merged**